worlds/shivers/__init__.py

```diff
@@ -138,6 +161,16 @@ def create_items(self) -> None:
             elif lobby_access_keys == 2:
                 self.multiworld.local_early_items[self.player]["Key for Front Door"] = 1
 
+        #Pot piece shuffle location:
+        if self.options.location_pot_pieces == 0:
+            self.options.local_items.value |= ({name for name, data in item_table.items() if data.type == "pot" or data.type == "pot_type2"})
```

Comment:
Little confused again about the parentheses

*low priority*

Reply:
Fixed